packages/components/CHANGELOG.md

```diff
@@ -1,9 +1,3 @@
-## 7.2.0 (Unreleased)
-
-### New Features
-
-- `ToggleControl` now accepts a `className` prop to merge with its default-assigned class name.
```

Comment:
By the way, on its own, it makes sense as well. We might want to add `className` prop to all reusable components located in `@wordpress/components` package.